pkg/sqlite/directory.go

```diff
@@ -201,6 +201,11 @@ func (d *DirectoryLoader) LoadPackagesWalkFunc(path string, f os.FileInfo, err e
 		return nil
 	}
 
+	if !strings.HasSuffix(f.Name(), ".yaml") && !strings.HasSuffix(f.Name(), ".json") {
```

Comment:
May wish to also add `.yml`

Reply:
Yup this was a conscious decision though as I was following along with another example from this project so I wasn't sure if it was a choice to not look include `.yml` https://github.com/operator-framework/operator-registry/blob/master/pkg/schema/schema.go#L301-L320 - happy to add it though.